spec/factories.rb

```diff
@@ -8,7 +8,7 @@
      name "St. Luis"
      responsable_name "Miss Faget"
      responsable_contact "123-456-789"
-     start_date "1.month"
-     end_date "1.month + 5"
+     start_date Date.today.next_year
+     end_date (Date.today.next_year + 5)
```

Comment:
Are these attributes required for a valid record?

Reply:
Also, if they are, I would rewrite this to be for example:
`1.month.from_now`, `2.months.from_now`, `1.year.from_now`...

Reply:
done in `822ec78`